mapper/cumulocity/c8y_translator_lib/src/lib.rs

```diff
@@ -820,22 +549,22 @@ mod tests {
     fn prefix_fn_removes_extra_chars() {
         let input = "薄いエッジ";
         assert_eq!(input.len(), 15);
-        assert_eq!(input_prefix(input, 1), "薄");
+        assert_eq!(tedge_json::input_prefix(input, 1), "薄");
     }
 
     #[test]
     fn prefix_fn_let_unchanged_short_inputs() {
         let input = "FØØ";
         assert_eq!(input.len(), 5);
-        assert_eq!(input_prefix(input, 4), input);
+        assert_eq!(tedge_json::input_prefix(input, 4), input);
     }
 
     use proptest::prelude::*;
 
     proptest! {
         #[test]
         fn prefix_doesnt_crash(input in "\\PC*") {
-            let _ = input_prefix(&input, 10);
+            let _ = tedge_json::input_prefix(&input, 10);
```

Comment:
This test of the private helper function should be moved where the function is defined.

Reply:
fixed